lesson02/assignment/css/_components.content.scss

```diff
@@ -14,4 +14,65 @@ html {
 
 h1 {
     color: $red;  
-}
\ No newline at end of file
+}
+
+.profile-card-container {
+    height: 40rem;
+    width: auto;
+    position: relative;
+    background-color: $bkgrdclr;
+    overflow: hidden;
+    box-shadow: 0 0 10px gray; // $gray doesn't work
+    border-radius: 10px;
+    margin: 3rem;
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    border: 2.5rem solid gray;
+}
+
+.profile-card-container {
+    &__card-img-area img {
+        width: 10rem;
+        border-radius: 50%;
+        margin-top: 1rem;
+        margin-bottom: .5rem;
+    }
+    &__card-name-area {
+        margin-bottom: .5rem;
+    }
+    &__card-name-area h3 {
+        margin: 0;
+    }
+    &__card-title-area {
+        font-size: small;
+    }
+}
+
+
+h4 {
+    margin-bottom: 0;
+    margin-top: .5rem;
+}
+
+h4::first-letter {
+    text-transform: uppercase;
+}
+
+p,a {
+    margin-top: .25rem;
+    margin-bottom: .5rem;
+    font-size: small;
+}
+
+.card-details-area {
+    background-color: #DFE7F1;
+    width: 12rem;
+    padding-top: 1rem;
+    padding-bottom: 1.5rem;
+    margin: .5rem auto;
+}
+
+.social-details{
+    padding-top: .5rem;
+}
```

Comment:
In your email you mentioned attempting media queries for responsive design, but I don't see any in here. You can try utilizing the mixin provided in `_settings.responsive.scss` or writing your own media query.

MDN tends to be the best place to read about anything web tech related: https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries